No such module 'Combine'

### Check List

Thanks for considering to open an issue. Before you submit your issue, please confirm these boxes are checked.

- [x] I have read the [wiki page](https://github.com/onevcat/Kingfisher/wiki) and [cheat sheet](https://github.com/onevcat/Kingfisher/wiki/Cheat-Sheet), but there is no information I need.
- [x] I have searched in [existing issues](https://github.com/onevcat/Kingfisher/issues?utf8=✓&q=is%3Aissue), but did not find a same one.
- [x] I want to report a problem instead of asking a question. It'd better to use [kingfisher tag in Stack Overflow](http://stackoverflow.com/questions/tagged/kingfisher) to ask a question.

### Issue Description
No such module 'Combine' when archiving a project with xcode11 branch using SwiftUI

#### What
Xcode produces "No such module 'Combine'" when archiving a project with xcode11 branch using SwiftUI.

#### Reproduce
- Create project
- Add swift package with xcode11 branch for SwiftUI
- Archive project

#### Other Comment
n/a

I tried using Carthage to get the latest Kingfisher version 5.8.2 and I got the compilation error:
```
 /Users/.../Carthage/Checkouts/Kingfisher/Sources/SwiftUI/KFImage.swift:27:8: error: no such module 'SwiftUI'
 import SwiftUI
        ^
 
 ** ARCHIVE FAILED **
```
The Cartfile looks like this:
```
github "onevcat/Kingfisher" ~> 5.0
```
which results in Cartfile.resolved:
```
github "onevcat/Kingfisher" "5.8.1"
```
Are there any options availble to resolve this (other than forking Kingfisher 😉)?

--- a/Sources/SwiftUI/ImageBinder.swift
+++ b/Sources/SwiftUI/ImageBinder.swift
@@ -24,9 +24,9 @@
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
 
+#if canImport(SwiftUI) && canImport(Combine)
 import Combine
 import SwiftUI
-
 #if !KingfisherCocoaPods
 import Kingfisher
 #endif
@@ -138,3 +138,4 @@ extension KFImage {
         }
     }
 }
+#endif

--- a/Sources/SwiftUI/KFImage.swift
+++ b/Sources/SwiftUI/KFImage.swift
@@ -24,8 +24,9 @@
 //  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 //  THE SOFTWARE.
 
-import SwiftUI
+#if canImport(SwiftUI) && canImport(Combine)
 import Combine
+import SwiftUI
 #if !KingfisherCocoaPods
 import Kingfisher
 #endif
@@ -205,4 +206,5 @@ struct KFImage_Previews : PreviewProvider {
         }
     }
 }
+#endif
 #endif

